lisa/transformers/parse_literal.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from ast import literal_eval
+from dataclasses import dataclass, field
+from typing import Any, Dict, List, Type
+
+from dataclasses_json import dataclass_json
+
+from lisa import schema
+from lisa.transformer import Transformer
+
+PARSE_LITERAL = "parse_literal"
```

Comment:
`literal_eval` may be more consistent. How about update existing `eval` transformer to use `literal_evel`?